### PR TITLE
fix db-delete to db-drop script name

### PR DESCRIPTION
**Pull request by bamtests**
#### Author checklist (Completed by original Author)
- [x] Contribution a good fit for the Rocky project? Title and Author MetaTags inserted ?
- [x] Is this a non-English contribution? 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Check that document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Basic Editorial Review)
- [x] 4th Pass (Detailed Editorial Review and Peer Review)
- [x] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by sspencerwire** [comment]
Hey thank you @bamtests for this fix. Unfortunately, we can't accept the fix for the Italian translation, as that must be done through the Crowdin interface. The source (English) document we can accept. Once the source is pulled back in, the translation interface will pick up the change in the source document in the Italian translation. Can you push edit your PR here to just include the fix for the English (source) document?  Then we can merge. Thank you again!

**Comment by sspencerwire** [comment]
@all-contributors please add @bamtests for contribution.

**Comment by sspencerwire** [comment]
@all-contributors please add @bamtests for content

File: docs/guides/containers/podman-nextcloud.it.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ eof
 Salvare e chiudere, quindi ripetere i passaggi con lo script per l'eliminazione dei database, se necessario:
 
 ```bash
-vi db-delete.sh
+vi db-drop.sh
```
Comment (SergeCroise) [inline docs/guides/containers/podman-nextcloud.it.md:197]:
```suggestion
vi db-delete.sh
```

Comment (SergeCroise) [reply docs/guides/containers/podman-nextcloud.it.md:197]: see https://crowdin.com/translate/rockydocs/320/en-it?filter=basic&value=0

**Comment by sspencerwire** [comment]
@bamtests Did you see my note regarding not including the Italian translation as part of this PR?  If you are unsure, I can pull this down and delete the Italian portion myself.

**Comment by bamtests** [comment]
@sspencerwire yes my apologies it took a minute to get back to github but that is done!  I appreciate adding me as a contributor for such a minor change lol...I may have more tho to earn it!

**Comment by sspencerwire** [comment]
Thanks!!!

**Pull request merged**